modules/@apostrophecms/area/ui/apos/components/AposAreaEditor.vue

```diff
@@ -422,12 +422,19 @@ export default {
       } else if (this.widgetIsContextual(name)) {
         return this.insert({
           widget: {
-            _id: cuid(),
             type: name,
             ...this.contextualWidgetDefaultData(name)
           },
           index
         });
+      } else if (!this.widgetHasInitialModal(name)) {
+        return this.insert({
```

Comment:
TODO: find why docId is null:

![image](https://user-images.githubusercontent.com/8301962/192333967-1e1eaf66-caa3-4ed5-b699-f93f9b6e5bd2.png)